src/main/java/com/minecolonies/coremod/configuration/ConfigurationHandler.java

```diff
@@ -35,130 +30,114 @@ public ConfigurationHandler()
 
     /**
      * Initializes the configuration.
-     * 
+     *
      * @param file File to read input from.
      */
     public static synchronized void init(final File file)
     {
         config = new Configuration(file);
-        
+
         loadConfiguration();
     }
-    
+
     /**
      * Reads all options from the file, and sets those parameters, and saves those in {@link Configurations}.
      * Saves file after reading.
      */
-    private static synchronized void loadConfiguration() 
+    private static synchronized void loadConfiguration()
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This method has 85 lines, which is greater than the 75 lines authorized. Split it into smaller methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS138)